presto-hive/src/test/java/io/prestosql/plugin/hive/HiveQueryRunner.java

```diff
@@ -178,8 +179,14 @@ public DistributedQueryRunner build()
                         .put("hive.storage-format", "TEXTFILE") // so that there's no minimum split size for the file
                         .put("hive.compression-codec", "NONE") // so that the file is splittable
                         .build();
+
+                Map<String, String> hiveTimestampNanosProperties = ImmutableMap.<String, String>builder()
+                        .putAll(hiveProperties)
+                        .put("hive.timestamp-precision", "NANOSECONDS")
+                        .build();
                 queryRunner.createCatalog(HIVE_CATALOG, HIVE_CONNECTOR_NAME, hiveProperties);
                 queryRunner.createCatalog(HIVE_BUCKETED_CATALOG, HIVE_CONNECTOR_NAME, hiveBucketedProperties);
+                queryRunner.createCatalog(HIVE_TIMESTAMP_NANOS_CATALOG, HIVE_CONNECTOR_NAME, hiveTimestampNanosProperties);
```

Comment:
It would be better to create this catalog only in these tests where it's used.

Reply:
I do not have access to common properties in the tests, and would need to clone those. Are you worried about extra test runtime it implies?

Reply:
I am concerned about two things

- startup time
- we always test with two, now three, catalogs, so we cannot be sure a single catalog works on its own

feel free to disregard